src/opnsense/service/templates/OPNsense/Proxy/squid.conf

```diff
@@ -3,38 +3,44 @@
 # Do not edit this file manually.
 #
 
-{# wrap http_port ssl bump configuration for reuse #}
-{% macro sslbump_httpsconfig(network, tags='') -%}
-  {% if helpers.exists('OPNsense.proxy.forward.sslbump') and OPNsense.proxy.forward.sslbump == '1' %}
-https_port {{network}}:{{OPNsense.proxy.forward.sslbumpport|default('3129')}} {{tags}} ssl-bump cert=/var/squid/ssl/ca.pem  dynamic_cert_mem_cache_size=10MB generate-host-certificates=on
-  {% endif %}
+{# wrap listener configuration for reuse #}
+{% macro listener_config(network, port='3129', tags='', protocol='') -%}
+{%     if protocol == 'ssl' %}
+{%         set listener_type = 'https_port' %}
+{%     else %}
+{%         set listener_type = 'http_port' %}
+{%     endif %}
+{%     if helpers.exists('OPNsense.proxy.forward.sslbump') and OPNsense.proxy.forward.sslbump == '1' %}
+{%         set sslparams = 'ssl-bump cert=/var/squid/ssl/ca.pem dynamic_cert_mem_cache_size=10MB generate-host-certificates=on' %}
```

Comment:
you may need to set sslparams to an empty string it not in sslbump mode, I haven't test this, but I'm not sure either it will like an uninitialised string.